.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
         args:
           - --target-version=py38
   - repo: https://gitlab.com/pycqa/flake8
-    rev: 4.0.1
+    rev: 3.9.2
```

Comment:
Why the downgrade?

Reply:
oh that's odd I ran `pre-commit autoupgrade`

Reply:
oh it's gitlab